doc/HOWTO/howto_setupAndDeployCustom.md

```diff
@@ -0,0 +1,144 @@
+# HOWTO: Set Up a Testnet and Deploy Custom Contracts
+
+## Goals 
+This HOWTO details the steps required to set up a local Stellar network, deploy a custom Soroban contract, and invoke the contract via CLI. It complements the [official guide][guide].
+
+## Process
+
+Prerequisites:
+- Make `cargo`, `rustup`, and `go` available. You will need a`rustc` version that is at least 1.74.
+- Make sure you have enough allocated space. If you're using the NixOS VM, you might need to add `services.logind.extraConfig = "RuntimeDirectorySize=2G";` to your configuration, rebuild and reboot the VM.
+- you probably want to add `~/.cargo/bin` to your PATH
+
+### Local Stellar Network
+Instead of building from source, Stellar provides a quickstart Docker image. You can call the below command from anywhere, and Docker will fetch the image.
+```sh
+docker run --rm -d \
+    -p 8000:8000 \
+    --name stellar \
+    stellar/quickstart:testing \
+    --testnet \
+    --enable-soroban-rpc
+```
+Note here that the [official guide][guide] would have you run this in interactive mode (`run ... -it ...`), but for just the network having it run in the background (`-d`) is less noisy and doesn't hijack the terminal.
+
+The above snippet deploys a testnet (`--testnet`), if you want a standalone network, replace that flag with `--standalone`.
```

Comment:
It looks like `--testnet` does not deploy a fresh net, but will join the official testnet?
https://github.com/stellar/quickstart/blob/master/README.md

Also, did you mean `--local` instead of `--standalone`?

Reply:
No, `--standalone` is what the [admin guide](https://developers.stellar.org/network/soroban-rpc/admin-guide) lists (and `--standalone` works, I've tested it).
RE testnet, you're correct, the text is sloppy, it should accurately read "testnet node".

Reply:
Okay, `--standalone` and `--local` are synonyms, but `--standalone` is deprecated:

https://github.com/stellar/quickstart/blob/696ccb26643a8d7052cfbb56f767356007950800/start#L126-L127

Reply:
Ah lovely, their docs are out of date.

Reply:
Your chance for a "stellar" PR!